### DISCUSSION TITLE
Ability to move, disable, or trim down driver telemetry in window

### DISCUSSION THREAD

**Discussion by blademan007** [f1multiviewer/issue-tracker#355] [Feature requests]
After Bahrain '23 GP, I noticed that Stroll roll hoop camera was aimed slightly downward. When live timing telemetry ticker is enabled it cuts off the top of the stream. In this case, or in general, it would be nice to have:

- Ability to move telemetry to bottom of window
- Ability to disable telemetry in a window altogether
- Ability to customize telemetry data displayed
- Ability to customize the above settings on each window. Currently, this is all or none.

**Comment by f1multiviewer** [comment]
Hey @blademan007, a few of these suggestions are already available:

- You can change the position of the driver telemetry (for all windows) in Settings
- You can press `T` to cycle through the 3 different driver headers (telemetry overlay, TLA + headshot, none)
- You can remove the speedometer/gears in settings, and you can enable/disable the best sectors. Further customizations might come in the future
- Ability to customize the position or the elements shown in the timing overlay aren't available, but I'm not sure if I would per-stream telemetry overlay overrides as a feature. The position I think could be an option though, so I'd be open to adding that at some point in the future.

**Comment by jeppeolesen** [comment]
Closing this as some of it has already been implemented, and some of it is covered by #227
